client-react/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./Footer.module.scss"
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+const Logo = ({children}: {children?: React.ReactNode}) => {return <div>{children}</div>}
+const Misc = ({children}: {children?: React.ReactNode}) => {return <div>{children}</div>}
+
+const Footer = ({children}: Props) => {
+  return(
+    <div className={styles['footer']}>
+      {children}
+    </div>
+  )
+}
+
+Footer.Item = Logo;
+Footer.Item = Misc;
```

Comment:
Why do we rewrite Footer.Item here to Misc?

Reply:
This component was removed in the latest commit